eng/common/CIBuild.cmd

```diff
@@ -1,2 +1,2 @@
 @echo off
-powershell -ExecutionPolicy ByPass -NoProfile -command "& """%~dp0Build.ps1""" -restore -build -sign -pack -publish -ci %*"
\ No newline at end of file
+powershell -ExecutionPolicy ByPass -NoProfile -command "& """%~dp0Build.ps1""" -restore -build -test -sign -pack -publish -ci %*"
```

Comment:
It looks like this PR reverted some changes from dotnet/wpf#8013.

Reply:
`eng/common/*` is 'controlled' by the arcade repo and shouldn't be checked into. Any changes there would be overwritten on the next arcade update. The right way to do that is to make an eng/cibuild.cmd which calls `build.ps1` the way you want and use that in CI scripts.

Reply:
@ThomasGoulet73 - We've moved the scripts here - https://github.com/dotnet/wpf/pull/8036/

Reply:
@pchaurasia14 Alright thanks, just wanted to make sure it didn't break anything since I remembered seeing manual changes to these files.